src/libraries/System.Net.Http/tests/FunctionalTests/SocketsHttpHandlerTest.cs

```diff
@@ -3143,20 +3143,23 @@ public sealed class SocketsHttpHandler_HttpClientHandler_Cancellation_Test_Http2
     }
 
     [ConditionalClass(typeof(HttpClientHandlerTestBase), nameof(IsMsQuicSupported))]
+    [Collection("NoParallelTests")]
```

Comment:
Nit:
```suggestion
    [Collection(nameof(NoParallelTests))]
```

Reply:
Technically, this is name of the collection, not name of the class right? 
```
[CollectionDefinition("NoParallelTests", DisableParallelization = true)]
```
It so happen we name them the same.

Reply:
Yes, but it still helps IMO by binding the string constant to a symbol name in the C# compilation. (Harder to make a typo assuming we may have more of these.)